content/root/quick-start-guide.textile

```diff
@@ -15,65 +17,84 @@ languages:
+p(tip#terms). The terms used in the guide such as 'connection', 'channel', 'message', 'publish' and 'subscribe' are described in the "glossary":/glossary.
+
+You're currently viewing the guide in <span lang="javascript">JavaScript</span><span lang="nodejs">Node.js</span><span lang="java">Java</span><span lang="ruby">Ruby</span><span lang="objc">Objective-C</span><span lang="python">Python</span><span lang="php">PHP</span><span lang="swift">Swift</span><span lang="csharp">C# .Net</span><span lang="flutter">Flutter</span>. If you would like to view it in another language, use the selector above to change it. 
+
+h2(#step-1). Step 1: Add the Ably Client Library SDK
 
 blang[javascript].
-  When using Ably Realtime or REST in a browser, simply include the Ably JavaScript library within the @<head>@ of your HTML page as follows:
+  The Ably Client Library SDK is available via CDN.
+
+  To get started with your project, reference the SDK within the @<head>@ of an HTML page:
 
-  bc[html]. <script src="//cdn.ably.io/lib/ably.min-1.js"></script>
+  bc[html]. <script src="//cdn.ably.com/lib/ably.min-1.js"></script>
 
-  You can also obtain the library as an "NPM module":https://www.npmjs.com/package/ably.
+  The SDK is also available as an "NPM module":https://www.npmjs.com/package/ably.
 
 blang[nodejs].
-  The Ably Realtime and REST library is obtainable as an "NPM module":https://www.npmjs.com/package/ably. Simply add to your @package.json@ file or install manually:
+  The Ably Client Library SDK is available as an "NPM module":https://www.npmjs.com/package/ably.
+
+  To get started with your project, add the SDK into your @package.json@ file or install it manually:
 
   ```[sh]
   npm install ably
   ```
 
 blang[ruby].
-  The Ably Realtime and REST library is obtainable as a "Ruby Gem":https://rubygems.org/gems/ably. Simply add to your @Gemfile@ or install manually:
+  The Ably Client Library SDK is available as a "Ruby Gem":https://rubygems.org/gems/ably.
+
+  To get started with your project, add the SDK to your @Gemfile@ or install it manually:
 
   ```[sh]
   gem install ably
   ```
 
 blang[python].
-  The Ably REST library is installable from "PyPI":https://pypi.python.org/pypi/ably as follows:
+  The Ably Client Library SDK is available from "PyPI":https://pypi.python.org/pypi/ably. Note that the SDK is only available with a REST interface for Python.
+
+  To get started with your project, install the SDK:
 
   bc[sh]. pip install ably
 
-  It can then be imported as:
+  Import the SDK with:
 
   bc[python]. from ably import AblyRest
 
 blang[php].
-  The Ably PHP library is available as a "composer package on packagist":https://packagist.org/packages/ably/ably-php and can be installed as follows:
+  The Ably Client Library SDK is available as a composer package on "packagist":https://packagist.org/packages/ably/ably-php. Note that the SDK is only available with a REST interface for PHP.
```

Comment:
I might be being pedantic here, so feel free to override this suggestion but ... can we say "... the PHP SDK only implements the REST interface" here? Otherwise it could be read as a "REST interface for PHP", when it's really a "REST interface to Ably"

Reply:
Worth being as clear as possible so I've updated this.